test/categories/test_adjective.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+from qual_id.categories.adjective import Adjective
+import random
+
+
+class TestAdjectives(unittest.TestCase):
+  def setUp(self):
+    self.adjective = Adjective()
+
+  def test__get_values__returns_list(self):
```

Comment:
The way unit tests for categories are written in this repo has changed recently. If you pull the latest changes and take a look at another test class, you'll see that it's been simplified a little